### PR TITLE
docs(chips): use consistent data structure in autocomplete example

### PR DESCRIPTION
**Pull request by banminkyoz**

### CODE REVIEW

**Comment by banminkyoz** [comment]
I signed it!

File: src/material-examples/chips-autocomplete/chips-autocomplete-example.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
       *ngFor="let fruit of fruits"
       [selectable]="selectable"
       [removable]="removable"
-      (remove)="remove(fruit)">
-      {{fruit.name}}
+      (removed)="remove(fruit)">
+        {{ fruit }}
```
Comment (jelbourn) [inline src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: Omit spaces in braces

Comment (banminkyoz) [reply src/material-examples/chips-autocomplete/chips-autocomplete-example.html]: i just remove them :D

**Comment by banminkyoz** [comment]
@jelbourn i tried to change the commit message but it not work :((. 
Do i have to change the first commit or last commit of this pull request.

**Comment by jelbourn** [comment]
You can use `git rebase -i upstream/master` (or whatever your remote is called instead of `upstream`) to do an interactive rebase, which will let you squash commits and also change the message

**Comment by banminkyoz** [comment]
My current branch is all mess up 😢 I'll create a new pull request on new branch. I will test to use git rebase more in my respositories. Seem it better than merge. It help the tree more clean 😄 .
Thank you so much @jelbourn

**Pull request closed without merging**